Add space, point and comma separators to all prices lists

vltrajvlien opened the pull request:
Good idea. I added all 3 separators to all the prices (and updated the lists names).

vltrajvlien committed:
Add space, point and comma separators to all prices lists

listsfordesign commented:
Hi, thanks for your contribution. Once you've merged this PR, we'll merge it to Lists master. Good job! 

PS: Have you seen the [new site](https://www.lists.design/)?

paucc committed:
Limits the building height and some format changes

This PR limits the height of the buildings between 1 and 15 floors, changes from `º` to `º` and the door numbers to `ª`, and fixes some typos when using `L'`.

paucc commented:
Sorry for the late merge, I didn't see the PR. 🙏
